lib/pbench/test/unit/server/test_requests.py

```diff
@@ -365,7 +367,7 @@ def test_upload_filetree_error(
         assert response.status_code == HTTPStatus.INTERNAL_SERVER_ERROR
 
         with pytest.raises(DatasetNotFound):
-            Dataset.attach(controller=self.controller, path=datafile.name)
+            Dataset.attach(path=datafile.name)
```

Comment:
I think I was just mechanically removing `controller` from attach calls; I may have left this deliberately due to the use of "path" rather than "name" ... but in this case it's rather bizarre, because it makes no sense to use `dataset.name` as a *path* in the first place.

Reply:
Oh, wait; this is `datafile.name`, which is a path. I suppose I could use `Tarball.stem(datafile)` with `query` or go ahead and use `attach(datafile)` ... but `attach` with `datafile.name` is a bit in-betweenish. Hmm.